share/doc/homebrew/External-Commands.md

```diff
@@ -31,7 +31,7 @@ A shell script for an command named `extcmd` should be named `brew-extcmd`. This
 	</tr>
   <tr>
     <td>HOMEBREW_CELLAR</td>
-		<td>The location of the Homebrew Cellar, where software is staged, by default <code>/usr/local/Cellar</code>.</td>
+		<td>The location of the Homebrew cellar, where software is staged. This will be <code>$HOMEBREW_PREFIX/Cellar</code> if that directory exists, or <code>$HOMEBREW_REPOSITORY/Cellar</code> otherwise.</td>
```

Comment:
Cellar should probably still be capitalised. It's a place (Like "Paris") rather than a "thing" in this block of text, I believe.